scipy/spatial/transform/_rotation.pyx

```diff
@@ -464,6 +464,10 @@ cdef inline double[:, :] _compose_quat(
 
     return product
 
+def compose_quat(p, q):
```

Comment:
not sure if there is a better way to expose the function to the transformation module

Reply:
Hmmm, also unclear to me